liferay-content-management/liferay-portal
# LPD-30143 PropsUtil does not resolve correctly in a string literal, so extract to variable

PropsUtil.get("default.admin.password") does not resolve correctly in a string literal

# What is this trying to solve?

[Link to ticket](https://issues.liferay.com/browse/LPD-30143)

Several poshi tests are failing with forbidden connection

# How am I fixing it?

Extract PropsUtil.get("default.admin.password") to a variable to let it resolve before using it in a string literal

# How can you verify that it works?

Deploy portal-web and check that poshi tests such as the following work

LocalFile.ExposeErcInWikiPageAttachment#CanReturenErcWithGetSiteWikiPagesByErc
LocalFile.ExposeErcInWikiPageAttachment#CanDeleteAttachmentByExternalReferenceCode
LocalFile.ExposeErcInWikiPageAttachment#CanReturenErcWithGetWikiPagesByWikiPageId
LocalFile.ExposeErcInWikiPageAttachment#CanReturenErcWithGetWikiPageAttachmentsByWikiPageId
